Version.cmake

```diff
@@ -1,5 +1,5 @@
 # To update the range-v3 version, from a *CLEAN* working directory, update the version numbers below.
 # This makefile will generate a new version.hpp, *AMEND THE MOST RECENT COMMIT*, and git-tag the commit.
 set(RANGE_V3_MAJOR 0)
-set(RANGE_V3_MINOR 1)
+set(RANGE_V3_MINOR 2)
```

Comment:
In my understanding, as long as the major version says '0', I can break anything at any time with impunity. Mwah-hah-ha!